TestApp/.babelrc

```diff
@@ -1,3 +0,0 @@
-{
```

Comment:
What is the purpose of this file change?

Reply:
Let me answer my own question - it's a change introduced by RN 0.58. (see the diff of blank new RN app at https://github.com/pvinis/rn-diff-purge/compare/version/0.57.8...version/0.58.0)

Reply:
Yes, you're right. Thank you for the link, it's very helpful.